web/packages/teleport/src/Locks/Locks.tsx

```diff
@@ -43,6 +44,20 @@ function getFormattedDate(d: string): string {
   }
 }
 
+function lockTargetsMatcher(
+  targetValue: any,
+  searchValue: string,
+  propName: keyof LockForTable & string
+) {
+  if (propName === 'targets') {
+    return targetValue.some(
+      ({ name, value }) =>
+        name.toUpperCase().includes(searchValue) ||
+        value.toUpperCase().includes(searchValue)
```

Comment:
We should also add the whole `name: value` string as a valid match that way if someone searches "user: req" they get results as opposed to having to only search by either `name` or `value`.

```
|| `${name}: ${value}`.toUppercase().includes(searchValue)
```

Reply:
Great idea. added.